setup-scripts/skynet-nginx.conf

```diff
@@ -17,12 +17,13 @@ server {
 	}
 
 	location /skynet/skyfile/ {
+		add_header Access-Control-Allow-Origin: *;
 		client_max_body_size 1000M; # make sure to limit the size of upload to a sane value
 		proxy_read_timeout 600;
 		# proxy this call to siad endpoint (make sure the ip is correct)
 		proxy_pass http://127.0.0.1:9980;
 		proxy_set_header Expect $http_expect;
-		proxy_set_header Access-Control-Allow-Origin: *;
+		proxy_hide_header Access-Control-Allow-Origin: *;
```

Comment:
I'm confused with this syntax. Why would `hide_header` take a value, I'd expect it'd take a key.
Also what's the reasoning, to me `add_header` in combination with `proxy_hide_header` is not really redundant, but in this case it is because the proxy header is set to the wildcard as well..

Reply:
What about doing `proxy_pass_header Access-Control-Allow-Origin`

Reply:
> What about doing `proxy_pass_header Access-Control-Allow-Origin`

`No 'Access-Control-Allow-Origin' header is present on the requested resource.`

Reply:
```suggestion
		proxy_hide_header Access-Control-Allow-Origin;
```

Reply:
> I'm confused with this syntax. Why would `hide_header` take a value, I'd expect it'd take a key.

my mistake, fixed